docs/connector-development/config-based/overview.md

```diff
@@ -209,5 +205,5 @@ pagination_strategy:
 The following connectors can serve as example of what production-ready config-based connectors look like
 
 - [Greenhouse](https://github.com/airbytehq/airbyte/tree/master/airbyte-integrations/connectors/source-greenhouse)
```

Comment:
no need to update the greenhouse link because the production connector is using low-code